pkg/appconsts/app_consts.go

```diff
@@ -19,12 +19,18 @@ const (
 	// SubtreeRootThreshold.
 	//
 	// The rationale for this value is described in more detail in ADR-013.
-	SubtreeRootThreshold int    = 64
-	TxSizeCostPerByte    uint64 = 10
-	GasPerBlobByte       uint32 = 8
-	MaxTxSize            int    = 8_388_608 // 8 MiB in bytes
-	TimeoutPropose              = time.Millisecond * 3500
-	TimeoutCommit               = time.Millisecond * 4200
+	SubtreeRootThreshold    int    = 64
+	TxSizeCostPerByte       uint64 = 10
+	GasPerBlobByte          uint32 = 8
+	MaxTxSize               int    = 8_388_608 // 8 MiB in bytes
+	TimeoutPropose                 = time.Millisecond * 8500
+	TimeoutProposeDelta            = time.Millisecond * 500
+	TimeoutPrevote                 = time.Millisecond * 3000
+	TimeoutPrevoteDelta            = time.Millisecond * 500
+	TimeoutPrecommit               = time.Millisecond * 3000
+	TimeoutPrecommitDelta          = time.Millisecond * 500
+	TimeoutCommit                  = time.Millisecond * 100
+	DelayedPrecommitTimeout        = time.Millisecond * 5200
```

Comment:
pls see new times for final review @rach-id and @cmwaters 

we can adjust these up until the mocha release, @rach-id was going to run a few more experiments to get estimates

I'm less sure on the 5200, but it would align w/ current block times for empty blocks while allowing the max amount of time